server/src/domain/library/library.service.spec.ts

```diff
@@ -130,7 +137,16 @@ describe(LibraryService.name, () => {
       );
     });
 
-    it('should not initialize when watching is disabled', async () => {
+    it('should not initialize watcher when watching is disabled', async () => {
+      configMock.load.mockResolvedValue(systemConfigStub.libraryWatchEnabled);
+      databaseMock.tryLock.mockResolvedValue(false);
```

Comment:
You probably don't need this and you probably should check that it is not even called if watching is turned off.

Reply:
Thanks for catching, I had put the lock = false statement in the wrong test

Reply:
I'm not completely sure that I fixed this one, please let me know if I didn't.